src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -214,7 +215,7 @@ internal abstract class TexlFunction : IFunction
 
         // This method is used for managing "x-ms-dynamic-values" and "x-ms-dynamic-schema" OpenApi extensions in connectors
         // https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions#use-dynamic-values
-        public virtual async Task<List<string>> GetConnectorSuggestionsAsync(CallNode callNode, int argPosition, CancellationToken cts)
+        public virtual async Task<List<RecordValue>> GetConnectorSuggestionsAsync(FormulaValue[] knownParameters, int argPosition, CancellationToken cts)
```

Comment:
>RecordValue [](http://example.com/codeflow?start=39&length=11)

RecordValue doesn't seem like right return type. 
Ie, these are not values that will flow into evaluator.